### ISSUE TITLE
Which model to use for genomic variants analysis?

### ISSUE THREAD

**Issue by nc1m** [jerryji1993/DNABERT#122]
Dear authors,

first thank you for your work!
I'm trying to recreate the results from section "3.5 Identifying functional genetic variants with DNABERT".
Which model should be used to compute the predictions?
Did you fine-tune a model on functional variants or used a pre-trained (DNABERT6) or other fine-tuned model (e.g. DNABERT-TF, DNABERT-Prom-300 or DNABERT-Prom-scan)?

Thank you for your time.

**Comment by vitoriastavis** [comment]
Hello, did you figure it out? I'm working on a project about variants.

Also, what did you use for labels in the input for prediction? In DNABERT/examples/ft/6/ the files have 1 and 0, but I'm not sure about the meaning. Maybe it's 1 for a condition and 0 for control, or 1 for SNP and 0 for none? The file DNABERT/SNP/examples/dev.tsv has different labels, such as 23, 104, 125.

Thanks in advance!
